.release-please-manifest.json

```diff
@@ -9,5 +9,7 @@
   "packages/relay": "0.0.16",
   "packages/sdk": "0.0.29",
   "packages/discovery": "0.0.6",
-  "packages/react-native-polyfills": "0.0.1"
+  "packages/react-native-polyfills": "0.0.1",
+  "packages/sds": "0.0.1",
```

Comment:
@adklempner adding `sds` to the release
it's absence was causing failures: release-please wasn't bumping `@waku/*` packages in it causing issue with proto package https://github.com/waku-org/js-waku/actions/runs/13945155395/job/39030577038?pr=2309